tests/test_game.py

```diff
@@ -65,16 +60,16 @@ def test_returns_current_player(standard_game):
     assert standard_game.get_current_player() == current_player_at_start
 
 
-def test_can_switch_current_player(standard_game):
+def test_play_turn_switches_the_current_player(standard_game):
```

Comment:
I like how you're testing expected behaviour here. I wonder if the test title could be more accurate, perhaps `test_player_two_has_turn_after_player_one`?

What is a `standard_game`?